packages/xstate-inspect/src/browser.ts

```diff
@@ -61,7 +61,7 @@ export function createDevTools(): XStateDevInterface {
 }
 
 const defaultInspectorOptions: InspectorOptions = {
-  url: 'https://statecharts.io/inspect',
+  url: 'https://stately.ai/viz',
```

Comment:
We'll be adjusting the public URL this week - probably to something like https://stately.ai/inspect . So until then, I would hold off merging this PR